conf/routes

```diff
@@ -80,7 +80,8 @@ POST   /researcher/v1/study/participants @controllers.StudyController.sendStudyP
 POST   /researcher/v1/study              @controllers.StudyController.updateStudyForResearcher
 
 # Researchers - Upload Schemas
-GET    /researcher/v1/uploadSchema/:schemaId  @controllers.UploadSchemaController.getUploadSchema(schemaId: String)
+GET    /researcher/v1/uploadSchema/byId/:schemaId  @controllers.UploadSchemaController.getUploadSchema(schemaId: String)
+GET    /researcher/v1/uploadSchema/forStudy   @controllers.UploadSchemaController.getUploadSchemasForStudy
```

Comment:
My main concern was that /uploadSchema/:schemaId and /uploadSchema/forStudy were too ambiguous with each other, so I wanted a /uploadSchema/byId/:schemaId to disambiguate. This way, if we want to fetch schemas with some other attribute, we could do /uploadSchema/byFoo/:foo. At the very least, in the extremely unlikely case someone wanted to name their schema "forStudy", there wouldn't be a problem, since we'd have separate namespaces.

Re: capitalization - I prefer to keep it camel-cased so we can read it. Unless you're arguing in favor of underscores (upload_schema) or dashes (upload-schema).